packages/gatsby/src/db/index.js

```diff
@@ -27,33 +27,14 @@ async function saveState() {
 const saveStateDebounced = _.debounce(saveState, 1000)
 
 /**
- * Sets up listeners so that once bootstrap has finished, all
- * databases save their state to disk. If we're in `develop` mode,
- * then any new event triggers a debounced save as well.
+ * Starts listening to redux actions and triggers a database save to
+ * disk upon any action (debounced to every 1 second)
  */
 function startAutosave() {
-  // During development, once bootstrap is finished, persist state on changes.
-  let bootstrapFinished = false
-  if (process.env.gatsby_executing_command === `develop`) {
-    emitter.on(`BOOTSTRAP_FINISHED`, () => {
-      bootstrapFinished = true
-      saveState()
-    })
-    emitter.on(`*`, () => {
-      if (bootstrapFinished) {
-        saveStateDebounced()
-      }
-    })
-  }
-
-  // During builds, persist state once bootstrap has finished.
-  if (process.env.gatsby_executing_command === `build`) {
-    emitter.on(`BOOTSTRAP_FINISHED`, () => {
-      saveState()
-    })
-  }
+  emitter.on(`*`, () => saveStateDebounced())
```

Comment:
should `startAutosave` also call `saveState` initially? Right now in develop we would start autosaving, but we potentially would never actually save if no action is emitted?

Reply:
Another good catch. Fixed in f6a254e